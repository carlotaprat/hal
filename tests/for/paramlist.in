for key, value in map:
    pass
