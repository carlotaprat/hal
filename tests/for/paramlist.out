digraph {

	ordering=out;
	ranksep=.4;
	bgcolor="lightgrey"; node [shape=box, fixedsize=false, fontsize=12, fontname="Helvetica-bold", fontcolor="blue"
		width=.25, height=.25, color="black", fillcolor="white", style="filled, solid, bold"];
	edge [arrowsize=.5, color="black", style="bold"]

  n0 [label="BLOCK"];
  n1 [label="FOR_STMT"];
  n1 [label="FOR_STMT"];
  n2 [label="PARAMS"];
  n2 [label="PARAMS"];
  n3 [label="key"];
  n4 [label="value"];
  n5 [label="FUNCALL"];
  n5 [label="FUNCALL"];
  n6 [label="map"];
  n7 [label="ARGS"];
  n8 [label="BLOCK"];
  n8 [label="BLOCK"];
  n9 [label="FUNCALL"];
  n9 [label="FUNCALL"];
  n10 [label="pass"];
  n11 [label="ARGS"];

  n0 -> n1 // "BLOCK" -> "FOR_STMT"
  n1 -> n2 // "FOR_STMT" -> "PARAMS"
  n2 -> n3 // "PARAMS" -> "key"
  n2 -> n4 // "PARAMS" -> "value"
  n1 -> n5 // "FOR_STMT" -> "FUNCALL"
  n5 -> n6 // "FUNCALL" -> "map"
  n5 -> n7 // "FUNCALL" -> "ARGS"
  n1 -> n8 // "FOR_STMT" -> "BLOCK"
  n8 -> n9 // "BLOCK" -> "FUNCALL"
  n9 -> n10 // "FUNCALL" -> "pass"
  n9 -> n11 // "FUNCALL" -> "ARGS"

}

